OWNERS

```diff
@@ -1,11 +0,0 @@
-approvers:
```

Comment:
I'm curious why the OWNERS and README.md are detected removed, they look still there on your PR branch?

Reply:
Move files first then it overrides OWNERS, will revert it.